### ISSUE TITLE
AI-designed “hyperfoods” can possibly help prevent cancer

### ISSUE THREAD

**Issue by jessonpagaduan** [khuyentran1401/machine-learning-articles#92]
## TL;DR
Using graph-based ML methods (graph representation learning or geometric deep learning), this interesting analysis identifies which foods, i.e. "hyperfoods", contain ingredients that might work in a similar fashion as medical drugs in beating cancer and other diseases.

### Article Link
https://towardsdatascience.com/hyperfoods-9582e5d9a8e4

### Author
Michael Bronstein

## Key Takeaways
* Tea and citrus fruits are examples of foods fulfilling both of these conditions: first, they contain multiple anti-cancer drug-like compounds identified by our ML model and confirmed from medical literature, and second, these compounds exert complementary anti-cancer effects.
* Besides the aforementioned tea and citruses, cabbage, celery, and sage are rather common, cheap, and broadly available hyperfoods. In a sense, this comes to no surprise, as many of these foods are advocated as healthy choices by nutrition experts and there is overwhelming evidence of their health benefits.

## Useful Code Snippets
N/A

## Useful Tools
N/A

## Comments/ Questions
Enjoyed reading this article. Nice application of machine learning.

**Comment by khuyentran1401** [comment]
Wow, this is really cool. I was not even aware of this technique. Thanks for posting this
